src/components/Publish/Actions/index.tsx

```diff
@@ -78,6 +81,15 @@ export default function Actions({
             <Button type="submit" style="primary" onClick={handleActivation}>
               Connect Wallet
             </Button>
+          ) : !isSupportedOceanNetwork ? (
+            <Tooltip
+              content={<AvailableNetworks />}
+              className={styles.errorTooltip}
+            >
+              <Button type="submit" style="primary" disabled>
+                Change Network
```

Comment:
Would go more info style on this one and rather show "Unsupported Network", and then add a tooltip indicator after it so it's less surprising when a tooltip opens on that button. Also to have an affordance on touch devices